scripts.js

```diff
@@ -1,7 +1,8 @@
 // grab elements
-const getInput = document.querySelector('input');
-const getList = document.querySelector('.list');
+const inputElement = document.querySelector('input');
+const listElement = document.querySelector('.list');
 
+// create new elements and add class list
 const addMove = document.createElement('i');
```

Comment:
These don't need to be globally declared, move them all inside the `addItem` function